ext/DimensionalDataMakie.jl

```diff
@@ -0,0 +1,450 @@
+module DimensionalDataMakie
+
+using DimensionalData, Makie
+using DimensionalData.Dimensions, DimensionalData.LookupArrays
+
+const DD = DimensionalData
+
+_paired(args...) = map(x -> x isa Pair ? x : x => x, args)
+
+
+# Shared docstrings: keep things consistent.
+
+const AXISLEGENDKW_DOC = """
+- `axislegendkw`: attributes to pass to `axislegend`.
+"""
+_keyword_heading_doc(f) = """
+# Keywords
+
+Keywords for $f work as usual.
+"""
+
+_dims_doc(f) = """
+- `dims`: A `Pair` or Tuple of Pair of `Dimension` or `Symbol`. Can be used to
+    specify dimensions that should be moved to the `X`, `Y` and `Z` dimensions
```

Comment:
One downside of using this same docstring for all methods is that, for example, the `Z` dimension doesn't exist in heatmaps and will throw an error if the user tries to use it.